pyvista/plotting/mapper.py

```diff
@@ -239,10 +229,38 @@ def configure_scalars_mode(
             n_colors,
             preference,
             interpolate_before_map,
-            rgb,
-            _custom_opac,
+            direct_scalars_color_mode,
         ):
-            """Configure scalar mode."""
+            """Configure scalar mode.
+
+            Parameters
+            ----------
+            scalars : numpy.ndarray
+                Array of scalars to assign to the mapper.
+
+            mesh : pyvista.Dataset
+                Dataset to assign the scalars to.
+
+            title : str
+                Name to assign the scalars.
```

Comment:
Seems like this does two things: if `title` is already the name of a data array then it will be fetched, otherwise the input `scalars` will be added to the mesh with name `title`. If I understand this correctly, this is a bit confusing and might need a mention here.

Reply:
Agreed.  Resolved in 129477c3c4867152eb1dfce0f50730aa61c3bc0c.